src/modules/font/freetype/TrueTypeRasterizer.cpp

```diff
@@ -121,11 +123,24 @@ GlyphData *TrueTypeRasterizer::getGlyphDataForIndex(int index) const
 	FT_Render_Mode rendermode = FT_RENDER_MODE_NORMAL;
 	if (hinting == HINTING_MONO)
 		rendermode = FT_RENDER_MODE_MONO;
+	else if (sdf == true)
```

Comment:
This can be simplified to `else if (sdf)`

Reply:
Okay, I took that simplification into account.